### PR TITLE
Feat: Added a link to updates in products

### PR DESCRIPTION
**Pull request by maanasb01**
In the `/products` route, each product now has a link to see all updates related to it.

### CODE REVIEW

File: src/pages/products/index.jsx [hunk 1]
```diff
@@ -156,6 +157,7 @@ const ProductIndex = () => {
                 url={"/products/dau"}
                 publicationDate={"13-11-2020"}
                 blog_slug={"/blog/tags/project/kosh"}
+                updates_slug={"/updates/tags/project/kosh"}
```
Comment (dennyabrain) [inline src/pages/products/index.jsx:160]: for both blog_slug and updates_slug make it /dau and not /kosh

Comment (maanasb01) [reply src/pages/products/index.jsx:160]: Sure, let me do that

Comment (maanasb01) [reply src/pages/products/index.jsx:160]: It appears that the /dau pages don't exist. Even in the Tags too, there is no dau tag, so the page for that project tag also doesn't exist. Also as all the project tags are being created based on the project field entry of all the pages ( blogs as well as updates), I think none of the mdx files has dau as a project entry.

image

These are the project tags for blogs, they don't have dau tag in them.

Comment (dennyabrain) [reply src/pages/products/index.jsx:160]: @maanasb01 its ok if they don't exist. in that case remove the "Read blogs" and "Read updates" section for DAU. will add them later after we've added blog and updates for DAU.

Comment (maanasb01) [reply src/pages/products/index.jsx:160]: Sure!

**Comment by maanasb01** [comment]
@dennyabrain Removed the blogs and updates links from the DAU in products.

**Review by dennyabrain — APPROVED** [review]
Looking good. Merging.

**Pull request merged**